docs/coverage.md

```
@@ -0,0 +1,217 @@
+# Install nightly toolchain
+rustup toolchain install nightly
+rustup component add llvm-tools-preview --toolchain nightly
+
+# Generate reports with branch coverage
+cargo +nightly coverage-branch       # HTML with branch data
+cargo +nightly coverage-branch-text  # Text with branch data
+```
+
+**Result**: Line, function, AND branch coverage data
+
+**Option 3: Use Both (Recommended)**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)